lib/score_tracker.rb

```diff
@@ -8,7 +8,7 @@ def self.has_player_won(board, character)
     is_a_diagonal_full(board, character)
```

Comment:
<img width="280" alt="screen shot 2017-08-30 at 8 00 10 pm" src="https://user-images.githubusercontent.com/14946902/29905001-bd97d1d8-8dc0-11e7-9dcf-a479ab16dd09.png">
There seems to be a bug somewhere in your logic. This should not be a draw.